zwave_js_server/model/command_class.py

```diff
@@ -51,3 +51,12 @@ def version(self) -> int:
     def is_secure(self) -> bool:
         """Return if the CommandClass is used securely on this node/endpoint."""
         return self.data["isSecure"]
+
+    def to_dict(self) -> dict[str, bool | int | str]:
+        """Create a dictionary from itself."""
+        return {
+            "id": self.id,
+            "name": self.name,
+            "version": self.version,
+            "is_secure": self.is_secure,
```

Comment:
I think we've decided that we want the keys to match the keys that are used to create the instance, ie camelCase format. This format will be used in the Home Assistant websocket commands too.